at_secondary/at_secondary_server/lib/src/server/at_secondary_config.dart

```diff
@@ -114,6 +118,14 @@ class AtSecondaryConfig {
     }
   }
 
+  static bool get testingMode {
+    var result = _getBoolEnvVar('testingMode');
```

Comment:
Note that this (environment) is how we will get the cicd secondaries to have testingMode set to true :  do `docker service update --force --env-add testingMode=true` on the cicd secondary containers